.github/zizmor.yml

```diff
@@ -0,0 +1,12 @@
+rules:
+  unpinned-uses:
+    config:
+      policies:
+        actions/*: ref-pin
+        helm/*: ref-pin
+        azure/*: ref-pin
+  dangerous-triggers:
+    ignore:
+      # We need the pull_request_target trigger here to allow us to add labels
+      # to PRs. We restrict the permissions within the job to "pull-requests: write"
+      - .github/workflows/pr-sizing.yaml
```

Comment:
I love the warning you added to `.github/workflows/pr-sizing.yaml`.

I think we should do some more stringent review of our usage of `pull_request_target`, it just doesn't feel like a it's false positive enough to ignore without further analysis. We have "Require approval for all external contributors" enabled on the repo, but zizmor makes me question if that is enough.

Do we want to do an "appsec" style review in this thread or would we prefer doing this in a new issue?